client/concordclient/src/concord_client.cpp

```diff
@@ -143,13 +143,20 @@ void ConcordClient::createGrpcConnections() {
 }
 
 void ConcordClient::checkAndReConnectGrpcConnections() {
+  bool needReconnection = false;
```

Comment:
Nit: Code style - `need_reconnection`.

Reply:
Thanks, @dartdart26, I will change this.